sonar-python-plugin/src/main/java/org/sonar/plugins/python/PythonProfile.java

```diff
@@ -56,12 +57,13 @@ private void loadActiveKeysFromJsonProfile(RulesProfile rulesProfile) {
   }
 
   public static Set<String> activatedRuleKeys() {
-    URL profileUrl = PythonProfile.class.getResource("/org/sonar/l10n/py/rules/python/Sonar_way_profile.json");
-    try {
+    String path = "/org/sonar/l10n/py/rules/python/Sonar_way_profile.json";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Refactor your code to get this URI from a customizable parameter. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1075)